CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing Guidelines
+
+## New Release
+
+1. Use a branch and PR targeting `master` (e.g. `devel`, `release/vM.m.p`, etc.)
+1. Keep the PR description up-to-date as things get merged into it
+1. Merge
+1. `git checkout master && git pull && npm version vM.m.p && git push --tags`
```

Comment:
if we follow this, we could either

- leave `release.yml` as-is, or
- change `release.yml` to be on tag push rather than on release published
  - this would allow us to also make the CI draft the GH release itself (e.g. copying commit comments and/or PR description) which reduces the amount of copy-pasting maintainers need to do

Reply:
> _this would allow us to also make the CI draft the GH release itself_

...and we would be again leaning towards the GitHub Releases automatic tagging model. 😄